.interface-design/system.md

```
@@ -95,6 +95,12 @@ Amber:   #f59e0b   (chart-4)
 Red:     #ef4444   (destructive / shadcn)
 ```
 
+### ARGUS overlay accent
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)